yaas/parsers/service.py

```diff
@@ -0,0 +1,23 @@
+# coding: utf-8
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+import inspect
+
+from .. import service
+from .. import config
+from .. import utils
+
+def define_subcommand(parent_parser):
```

Comment:
It's not idempotence, but I don't think it's bike-shedding either. But ya... it's minor.